piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -117,6 +117,32 @@ def is_operator_installed(self, operator_name: str, operator_namespace: str) ->
         """
         return self.csv.get_cluster_service_version(operator_name, operator_namespace) is not None
 
+    def get_version_of_operator(self, operator_name: str, operator_namespace: str) -> str:
```

Comment:
```suggestion
    def get_version_of_operator(self, operator_name: str, operator_namespace: str) -> Optional[str]:
```
the annotated return type should an `Optional[str]` which you can import from the `typing` module